cli/script/command-parser.ts

```diff
@@ -526,10 +525,6 @@ function createCommand(): cli.ICommand {
 
             case "logout":
```

Comment:
I don't think I can remove this case, as we still need to construct the command with the right enum value here.